src/webcore/macros.rs

```diff
@@ -473,6 +477,17 @@ macro_rules! _js_impl {
 ///
 /// println!( "2 + 2 = {:?}", result );
 /// ```
+///
+/// ## No Return
+///
+/// This will moderately increase performance when there is no return value.
```

Comment:
Hmm... I'm not sure if `moderately` is appropriate here, as I haven't really benchmarked it so I would rather not oversell it. (:

Perhaps a phrasing such as this would be better?

> If you don't need to return a value from your snippet you can add a `@(no_return)` attribute to slightly improve performance.

Side note: once procedural macros stabilize I'm planning on automatically derive whenever a given snippet returns a value or not; this attribute will then become unnecessary.